src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -194,20 +177,10 @@ export const NavSearchBar: React.FC<NavSearchBarProps> = ({
 
       {/* @debt use only one PortalModal instance with state controlled with redux */}
       <PortalModal
-        show={isDefined(selectedRoom)}
-        portal={selectedRoom}
-        venue={selectedRoomVenue}
+        show={isDefined(selectedPortal)}
+        portal={selectedPortal}
         onHide={hidePortalModal}
       />
-
-      {/* @debt use only one EventModal instance with state controlled with redux */}
```

Comment:
Do we no longer need this modal? If so is there any confirmation for it?

Reply:
If it's not needed, then let's remove the modal files as well

Reply:
not sure, Events were removed from the search list because we changed the way we fetch them, so I guess it was faster to remove that functionality rather than fix it everywhere. But I do not know if we're going to bring them back. 

@colinhowe, will we bring the Experiences (aka WoldEvents) back to the search results? or we won't need an EventModal anymore?

Reply:
removed it: https://github.com/sparkletown/sparkle/pull/2758/commits/9e3f251fe8c55fbfdcc0c7dda3ce905572d77e2f